options/posix/generic/grp-stubs.cpp

```diff
@@ -29,23 +29,25 @@ namespace {
 	}
 
 	bool extract_entry(frg::string_view line, group *entry) {
-		__ensure(!entry->gr_name);
-		__ensure(!entry->gr_mem);
```

Comment:
These assertions protect against a memory leak. We should call `clear_entry` at the appropriate place in the caller to prevent it but keep these assertions.